### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
This updates dependencies based on the list found [here](https://david-dm.org/cerner/terra-dev-site).

Closes [#170](https://github.com/cerner/terra-dev-site/issues/170).

### Additional Details
The following dependencies are updated:
[fs-extra](https://www.npmjs.com/package/fs-extra) to v8.0.0
[babel-eslint](https://www.npmjs.com/package/babel-eslint) to v10.0.1
[danger](https://www.npmjs.com/package/danger) to v8.0.0

babel-eslint [v9.0.0](https://github.com/babel/babel-eslint/releases/tag/v9.0.0) includes a change to account for the new [decorators proposal](https://babeljs.io/blog/2018/09/17/decorators) in babel v7. This is not a breaking change for terra, but is worth noting for future reference.

### CODE REVIEW

**Comment by mjhenkes** [comment]
@amichaelparker I'm making this ready for review to get travis to build

File: package.json [hunk 1]
```diff
@@ -113,11 +113,11 @@
     "@babel/plugin-transform-object-assign": "^7.2.0",
     "@babel/preset-env": "^7.4.4",
     "@babel/preset-react": "^7.0.0",
-    "babel-eslint": "^8.2.1",
+    "babel-eslint": "^10.0.1",
     "babel-jest": "^24.8.0",
     "check-installed-dependencies": "^1.0.0",
     "core-js": "^3.1.3",
-    "danger": "^3.7.14",
+    "danger": "^8.0.0",
```
Comment (emilyrohrbough) [inline package.json]: should we remove danger and add the github pr bot since this isn't a mono-repo? @bjankord

Comment (bjankord) [reply package.json]: Yeah, it would be one less dependency we have to worry about. 👍

Comment (amichaelparker) [reply package.json]: Removed: [c1dabdf](https://github.com/cerner/terra-dev-site/pull/171/commits/c1dabdf8e7c8d0869db48c27572985e3ae9e9cb20)

**Pull request merged**